packages/contracts/test/libraries/StringUtils/StringUtils_hexToBytes.t.sol

```diff
@@ -9,35 +9,42 @@ contract StringUtils_HexToBytes_Test is StructHelper {
         super.setUp();
     }
 
+    /// forge-config: default.allow_internal_expect_revert = true
```

Comment:
Inlining this config change is considered [safe](https://book.getfoundry.sh/cheatcodes/expect-revert#description) so long as it is not being used when there are multiple internal functions is a single test function

https://book.getfoundry.sh/guides/v1.0-migration#expect-revert-cheatcode-disabled-on-internal-calls-by-default